langchain/vectorstores/matching_engine.py

```diff
@@ -0,0 +1,373 @@
+"""Vertex Matching Engine implementation of the vector store."""
+
+import json
+import logging
+import time
+import uuid
+
+from google.cloud import aiplatform
```

Comment:
all google imports should happen within constructor (or other class / instance methods). see `langchain/document_loaders/googledrive.py` for an example